test/unit/annotation_spec.js

```diff
@@ -231,6 +231,243 @@ describe('annotation', function() {
 
       expect(annotation.color).toEqual(new Uint8ClampedArray([0, 0, 0]));
     });
+
+    it('should parse Contents', function () {
```

Comment:
This unit test can be removed now as it was included in the PR you rebased upon.